neard/src/runtime/errors.rs

```diff
@@ -0,0 +1,107 @@
+use easy_ext::ext;
+
+use near_chain::near_chain_primitives::error::QueryError;
+
+#[ext(FromStateViewerErrors)]
```

Comment:
Let's just use it by absolute path since it is the only place, and we don't need to import it. It also follows the style we already have

```suggestion
#[easy_ext::ext(FromStateViewerErrors)]
```